spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-page-views.spec.js

```diff
@@ -277,6 +277,30 @@ describe('Google Tag Manager page view tracking', function () {
     expect(window.dataLayer[0]).toEqual(expected)
   })
 
+  it('handles removing hyphens and downcases from the publishing government meta tag', function () {
+    createMetaTags('ga4-publishing-government', '2005-To-2010-Labour-GOVERNMENT')
+    expected.page_view.publishing_government = '2005 to 2010 labour government'
+    GOVUK.analyticsGa4.analyticsModules.PageViewTracker.init()
+    expect(window.dataLayer[0]).toEqual(expected)
+
+    createMetaTags('publishing-government', '2005-To-2010-Labour-GOVERNMENT')
+    expected.page_view.publishing_government = '2005 to 2010 labour government'
+    GOVUK.analyticsGa4.analyticsModules.PageViewTracker.init()
+    expect(window.dataLayer[0]).toEqual(expected)
```

Comment:
In both cases in this test (and below) you're checking `window.dataLayer[0]`, should the second one test `window.dataLayer[1]`?

Reply:
@andysellick Good spot - fixed!